apps/prairielearn/src/cron/index.ts

```diff
@@ -162,11 +162,11 @@ export async function init() {
 
   logger.verbose(
     'initializing cron',
-    _.map(jobs, (j) => _.pick(j, ['name', 'intervalSec'])),
+    jobs.map(({ name, intervalSec }) => ({ name, intervalSec })),
   );
 
   const jobsByPeriodSec = _.groupBy(jobs, 'intervalSec');
```

Comment:
Once we upgrade to Node 22, we'll get [native `Object.groupBy`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/groupBy) 🎉